webapp/views/launch/script.js

```diff
@@ -519,6 +522,28 @@ function buttonElement(action) {
   return $(`<button type="button" class="btn btn-primary">${action}</button>`);
 }
 
+/**
+ * create a table element from data
+ * @param {Array} data an array of identically structured js objects 
+ * @param {string} className className of the table
+ */
+function tableElement(className, data) {
+  let html = `<table class=${className}><tr>`
+  Object.keys(data[0]).forEach(field => {
+    html += `<th>${field}</th>`
+  })
+  html += '</tr>'
+  data.forEach(entry => {
+    html += '<tr>'
+    Object.keys(entry).forEach(field => {
+      html += `<td>${entry[field]}</td>`
+    })
+    html += '</tr>'
+  })
+  html += '</table>'
+  return html
```

Comment:
Again, use jQuery. Or define the table skeleton in the Pug template so it only needs to be filled with values.

Reply:
fixed